WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1284,6 +1290,31 @@ public void onDismiss(DialogInterface dialog) {
         return view;
     }
 
+    private boolean isValidIpOrRange(String entry) {
+        if (TextUtils.isEmpty(entry)) {
+            return false;
+        }
+
+        // remove whitespace
+        entry = entry.trim();
+        entry = entry.replaceAll(" ", "");
```

Comment:
Heh, that's what I thought at first too! `trim` only drops leading and trailing whitespace, so we need to replace in case there are spaces in the middle of the string.